RFS/src/main/java/com/rfs/cms/ReactorHttpClient.java

```diff
@@ -0,0 +1,97 @@
+package com.rfs.cms;
+
+import com.rfs.common.ConnectionDetails;
+import com.rfs.common.RestClient;
+import io.netty.handler.codec.http.HttpMethod;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import reactor.core.publisher.Mono;
+import reactor.netty.ByteBufMono;
+import reactor.netty.http.client.HttpClient;
+import reactor.netty.http.client.HttpClientResponse;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+@Slf4j
+public class ReactorHttpClient implements AbstractedHttpClient {
```

Comment:
Why use apache and reactor, why can't we just use one?

Reply:
I haven't gotten the ReactorHttpClient to work yet.  I'll add a comment & a jira task.